CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+11.4.0
+-----
+* Modularizes durable session storage
+* Introduces per-user token support and user session management (beta)
```

Comment:
What does "beta" mean here?

Reply:
I think this means that it requires a beta flag right now